src/ItemDetail.js

```diff
@@ -0,0 +1,27 @@
+import React, {useState, useEffect} from 'react';
+import './App.css';
+// import {Link} from 'react-router-dom';
+
+function Item() {
+
+  useEffect(() => {
+    fetchItem();
```

Comment:
A couple of syntax errors here which are not throwing runtime errors due to Babel.

* calling `fetchItem` before `fetchItem` is defined
* Two variables called `fetchItem`